classes/rootfs-image-generator.bbclass

```diff
@@ -82,6 +82,9 @@ do_create_rootfs_image() {
         init_ostree_repo
     fi
 
+    ROOTFS_FULL_PATH=$(realpath ${ROOTFS_SOURCE_DIR})
```

Comment:
It should be done if SE linux enabled. Right?

Reply:
Yes, you're right. Should I add checking the presence SELinux in DISTRO_FEATURES?

Reply:
I've added checking.